### DISCUSSION TITLE
Feature Request: Half grid bin generation support

### DISCUSSION THREAD

**Discussion by deivuh** [Le0Michine/FusionGridfinityGenerator#68] [Ideas]
I recently came across a few custom Gridfinity bins that take half a grid, I used to consider using half a grid as a bad idea, but I just realized that this can actually save quite some valuable space and also this is preferred over generating smaller grids, to keep the standard / universal Gridfinity values unaltered.

Maybe something like allowing `x.5` as an input when generating bins?

**Comment by Le0Michine** [comment]
Hi, not sure if I understand the idea correctly, maybe some links/pictures would help. Right now you can specify units dimensions as 21mm instead of default 42mm which should give you bins with quarter size squares at the bottom allowing to position them half way on the standard grid.
